setup/git/000_EnvSetup.rb

```diff
@@ -1,70 +1,44 @@
 test_name "Setup environment"
 
-step "Ensure Git and Ruby"
-
 extend Beaker::DSL::InstallUtils
 
+step "Install git and tar"
+
 PACKAGES = {
   :redhat => [
     'git',
-    'ruby',
-    'rubygem-json',       # invalid on RHEL6
-    'rubygem-io-console', # required for Fedora25 to bundle install
-    'rubygem-rdoc'        # required for Fedora25 to install gems
   ],
   :debian => [
     ['git', 'git-core'],
-    'ruby',
-  ],
-  :debian_ruby18 => [
-    'libjson-ruby',
-  ],
-  :solaris_11 => [
-    ['git', 'developer/versioning/git'],
   ],
   :solaris_10 => [
     'coreutils',
-    'curl', # update curl to fix "CURLOPT_SSL_VERIFYHOST no longer supports 1 as value!" issue
     'git',
-    'ruby19',
-    'ruby19_dev',
-    'gcc4core',
+    'gtar',
   ],
   :windows => [
     'git',
-    # there isn't a need for json on windows because it is bundled in ruby 1.9
+  ],
+  :sles => [
+    'git',
   ],
 }
 
-# override incorrect FOSS (git) defaults from Beaker with AIO applicable ones
-#
-# Remove after PUP-4867 breaks distmoduledir and sitemoduledir into individual
```

Comment:
Guessing the comment about [PUP-4867](https://tickets.puppetlabs.com/browse/PUP-4867) and the related code became irrelevant due to other refactoring?

Reply:
previously we were using the moduledirs defined in the foss defaults file, but we're not anymore with https://github.com/puppetlabs/beaker-puppet/pull/52/files#diff-7e0996ffe23c8cf4e3bc5558f32f7aa8L24

This will load the aio defaults, so we don't need to keep overwriting them here. I'm actually not sure how PUP-4867 relates to this, other than they both deal with the moduledir.... but by changing which defaults are loaded for git testing, we no longer have to override distmoduledir or sitemoduledir